tree/dataframe/src/RDataFrame.cxx

```diff
@@ -383,6 +389,24 @@ ROOT::EnableImplicitMT();
 ~~~
 Simple as that. More details are given [below](#parallel-execution).
 
+## <a name="collections"></a> Working with collections
+
+RDataFrame reads collections as the special type RVec (e.g. a branch containing an array of floating point numbers can
+be read as a `RVec<float>`). C-style arrays (with variable or static size), `std::vector`s and most other collection
+types can be read this way. When reading ROOT data, column values of type `RVec<T>` perform no copy of the underlying array.
+
+RVec is a container similar to `std::vector` but it offers a rich interface to operate on the array elements in a
+vectorised fashion, similar to Python's NumPy arrays.
+
+For example, to fill a histogram with the `pt` of selected particles for each event, Define() can be used to create
+a column that contains the desired array elements as follows:
+
+~~~{.cpp}
+# h is filled with all the elements of `good_pts`, for each event
+h = df.Define("good_pts", "pt[pt > 0]").Histo1D("good_pts")
```

Comment:
Tricky! Typically we use the `good_pts` object as a mask given the result of `pt > 0`. This may clash with other docs?

Reply:
Fair enough, I cannot come up with a better naming.